ansible_collections/ds/ansible_ds/playbooks/dscreate.yml

```diff
@@ -1,47 +1,38 @@
-- name: Playbook to create a new instance if it does not alreadfy exist
+---
+- name: Playbook to create a iset of DS instances
   hosts: localhost
-  collections:
-    - ds.ansible_ds
-
-  roles:
-  - role: ansible_collections/ds/ansible_ds/playbooks/roles/install_dspkg
-    state: present
 
   vars:
-    instdict:  "{{
-        {
-            instance:
-              {
-                'state': 'present',
-                'port': port,
-                'secure_port': secure_port,
-                'root_password': rootpw,
-                'backends': bedict,
-              }
-        }
-    }}"
-
-  tasks:
-    - name: load encrypted variables
-      include_vars: "vault.yml" 
-
-    - name: load the instance configuration variables from dscreat_config.yml
-      include_vars: "dscreat_config.yml"
-
-    - name: populate the backends dict variable
-      set_fact:
-        bedict: "{{ bedict|default({}) | combine( {item.key: { 'suffix': item.value, 'state': 'present'} } ) }}"
-      loop: "{{ backends|dict2items }}"
-
-    - name: debug instdict
-      debug:
-        var: instdict
-
-    - ds.ansible_ds.ds_update:
-       content:
-          state: present
-          prefix:  "{{ lookup('env', 'PREFIX') | default('', True) }}"
-          instances: "{{instdict}}"
-      tags: vars
+    # In a real deployment, variables should be customized
+    # In this playbook all the variables are commented out because
+    # they are provided by the calling test playbook
```

Comment:
I think it makes sense to add an example variable here (as a working example which can be customized later).
So if the administrator wants to test the playbook - he can run it out of the box.

And then he will customize the variables for his real deployment.
What do you think?